packages/backend/src/tokens/authenticateContext.ts

```diff
@@ -0,0 +1,88 @@
+import { constants } from '../constants';
```

Comment:
These classes will be moved to a different dir when I wrap up my work on clerkMiddleware